ts/nni_manager/common/experimentStartupInfo.ts

```diff
@@ -126,3 +126,8 @@ export function getDispatcherPipe(): string | null {
 export function getAPIRootUrl(): string {
     return getExperimentStartupInfo().apiRootUrl;
 }
+
+export function getPrefixUrl(): string {
+    const prefix = getExperimentStartupInfo().urlprefix === ''?'':`/${getExperimentStartupInfo().urlprefix}`;
```

Comment:
Please add spaces around operators to make the style in sync with other codes.

Reply:
why not add lint for this format in next release?

Reply:
fixed.